### PR TITLE
docs: add defaultBrowser config option

### PR DESCRIPTION
**Pull request by alexsch01**
For https://github.com/cypress-io/cypress/pull/30517

### CODE REVIEW

**Review by MikeMcC399 — COMMENTED** [review]
Wouldn't it make more sense to put this in the https://docs.cypress.io/app/references/configuration#Browser section?

**Review by MikeMcC399 — COMMENTED** [review]
What about https://docs.cypress.io/app/references/module-api#Options ?

Shouldn't there be an entry there as well?

- (I may have been misled due to the comment in https://github.com/cypress-io/cypress/pull/30517#pullrequestreview-2430146621)

**Comment by alexsch01** [comment]
> Wouldn't it make more sense to put this in the https://docs.cypress.io/app/references/configuration#Browser section?

changed, I think that makes sense

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Linting is failing, so 

```shell
npm run lint:fix
```

needs to be run

File: docs/app/references/configuration.mdx [hunk 1]
```diff
@@ -146,6 +146,7 @@ For more options regarding screenshots, view the
 
 | Option                  | Default | Description                                                                                                                                                                                                                                                                                                                                           |
 | ----------------------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `defaultBrowser`        | `null`  | The default browser to launch if the "--browser" command line option is not provided. This option only affects the first browser launched; changing this option after the browser is already launched will have no effect.                                                                                                                            |
```
Comment (mschile) [inline docs/app/references/configuration.mdx:149]:
```suggestion
| `defaultBrowser`        | `null`  | The default browser to launch if the "--browser" command line option is not provided. This option only affects the first browser launch; changing this option after the browser is already launched will have no effect.                                                                                                                              |
```

**Review by mschile — APPROVED** [review]

**Pull request merged**